owncloud/owncloud.py

```diff
@@ -864,6 +1187,31 @@ def __make_ocs_request(self, method, service, action, **kwargs):
         res = self.__session.request(method, self.url + path, **attributes)
         return res
 
+    def __make_single_request(self, method, path, **kwargs):
```

Comment:
Did you comment on the wrong block ? I don't see "make_single_request" used anywhere.
The "create_user" uses "make_ocs_request" instead.